lua/lvim/core/alpha/startify.lua

```diff
@@ -0,0 +1,17 @@
+local M = {}
+
+function M.config()
+  local startify = nil
+
+  return startify
```

Comment:
eh, what is going on here 😅 
`return nil` or what?

Reply:
I wanted to add the integration, but I got tired yesterday 😄